### PR TITLE
add code commenting

### PR DESCRIPTION
**Pull request by markedwardmurray**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,18 +86,21 @@ And now it will hold the value `4`.
 
 Variables can be of many types: strings, arrays, numbers, dictionaries, and many more. The types of variables which refer only to a value (such as `NSInteger`) are referred to as "**primitives**". This is because, unlike the other variable types called "**objects**", primitives are unable to perform behaviors on the values that they reference. Subsequently, an object is a variable which can refer to one or many values that it can itself perform actions upon (these actions are called "**methods**"). 
 
-A string is a type of object which is used to store and manipulate text. In Objective-C, the base string class is `NSString`. We can create a string variable in much the same way as creating an integer variable.
+A string is a type of object which is used to store and manipulate text. While the string literal can be used in conjunction with `NSLog()` to print directly to the console:
```
Comment (misterfifths) [inline README.md]: Hmm.. not a complete sentence. Maybe: "We've already seen string literals used in conjunction with `NSLog()` to print directly to the console:..."

File: README.md [hunk 2]
```diff
@@ -87,18 +86,21 @@ And now it will hold the value `4`.
...
-
-This is because `NSString` is an object type (we'll cover in depth what this means in the next topic). Objects require a pointer reference (the `*`) in their declaration because, well, they point to a set of values at a specific memory address in the computer's RAM. However, `NSInteger` is a primitive, or "data type", and is itself a value, rather than a pointer to a value or set of values.
+We'll cover in detail the distinction between objects and primitives in a later topic on Object-Oriented-Programming. For now, just recognize that objects hold values and can change them through behaviors called "methods", while primitives just hold a value. Because of this, primitives **do not** employ the `*` in the their declaration syntax.
 
 **Note:** *Entering* `NSInteger *` *will cause the compiler to generate an* `invalid integer to pointer conversion` *warning. The same goes for declaring any of the data types. This syntax does serve a purpose so the language permits it, but your application won't do what you expect.*
```
Comment (misterfifths) [inline README.md]: Hmm that first sentence is a little misleading. "Assigning an integer to a variable of type `NSInteger *` will cause the compiler..."

File: README.md [hunk 3]
```diff
@@ -110,4 +112,63 @@ NSUInteger
...
+
+```objc
+/*
+A block comment will comment-out all the text between 
+its opening and closing symbols. They use a slash and star 
+to open, and a star and slash to close.
+*/
+```
+
+Commenting has its usefulness but in general should be treated as a last resort. Learn to keep your commenting to a minimum and use the code itself to document your intent.
```
Comment (misterfifths) [inline README.md]: I'd go a little less strongly on this. "Commenting is useful for documenting non-obvious behavior, but it is not a substitute for well-written code. Learn to keep your commenting..."

**Comment by misterfifths** [comment]
Looking good! Comments made.

**Comment by markedwardmurray** [comment]
closes #5

**Pull request merged**